scripts/reset-user-testing.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+
+# shellcheck disable=SC1090
+source "${SCRIPT_DIR}/common.sh"
+
+check_aws_account_used prod
```

Comment:
I don't really want to encourage people to use this script for manual testing. Are you sure these changes would be valuable in the longer term, or did you think of them for a more convenient review?